Carthage.xcodeproj/xcshareddata/xcschemes/carthage.xcscheme

```diff
@@ -77,6 +77,18 @@
             isEnabled = "YES">
          </CommandLineArgument>
       </CommandLineArguments>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "DEVELOPER_DIR"
+            value = "${DEVELOPER_DIR}"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "TOOLCHAINS"
+            value = "${TOOLCHAINS}"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+      </EnvironmentVariables>
```

Comment:
I added this for supporting the case that running tests through Xcode 8 but Xcode 7.3.1 is selected for `xcodebuild` (Command Line Tools) and the opposite. If this is added and enabled, the `xcodebuild` commands executed in Carthage tests would be the same version with the Xcode a user is launching.

Does that makes sense or should we remove it?